formats/polus-ome-zarr-converter-plugin/src/main.py

```diff
@@ -17,12 +16,12 @@
 # TODO: In the future, uncomment this to convert files to the platform file type
 # FILE_EXT = os.environ.get('POLUS_EXT',None)
 # FILE_EXT = FILE_EXT if FILE_EXT is not None else '.ome.tif'
-FILE_EXT = ".ome.zarr"
+#FILE_EXT = ".ome.zarr"
```

Comment:
The easiest way to incorporate the changes you want to make here are to leave `FILE_EXT` as is, then add `FILE_EXT = fileExtension` in the `image_converter` function.

This will also make this plugin significantly easier to modify later when we want to make this a standard file format converter, since we can just remove the `FILE_EXT = fileExtension` line from the function.

Reply:
@hsidky made a good comment in a discussion we had, and you should ignore this comment.

Reply:
I defined variable FILE_EXT = fileExtension in the image_converter function which will make later modifications easier to this plugin